pyproject.toml

```diff
@@ -6,6 +6,9 @@ authors = ["andreyPromaster <and45.rew@ya.ru>"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
+Flask = "^2.0.2"
+pydantic = {extras = ["email", "dotenv"], version = "^1.8.2"}
```

Comment:
Why extras?

Reply:
`pydantic[dotenv]` provides optional  dotenv file support with Settings. `email `is redundant. i will remove it/